src/microbe_stage/CompoundCloudPlane.cs

```diff
@@ -412,8 +412,7 @@ public void GetCompoundsAt(int x, int y, Dictionary<Compound, float> result)
                 break;
 
             float amount = HackyAddress(Density[x, y], i);
-            if (amount > 0)
-                result[Compounds[i]] = amount;
+            result[Compounds[i]] = amount;
```

Comment:
Why was this change made?

Reply:
I changed the code, so that you can reuse the same instance of the Dictionary to not always create a new Dictionary on every frame.
So this is needed to zero the value which maybe already in the Dictionary from the previous query.

Reply:
Why not clear the dictionary then (in the hover info)? This changes the functionality here from "what clouds there are here" to "what cloud types are supported", and I think that kind of assumption change needs more careful investigation regarding the overall code structure that uses the clouds.

Reply:
reverted and cleared dictionary.